MultipleFileUpload/Model/Dibi/Queues.php

```diff
@@ -45,38 +44,24 @@ function initialize()
 
 	}
 
-
-	// <editor-fold defaultstate="collapsed" desc="Database functions">
-
-	/**
-	 * Gets dibi connection
-	 * @return \DibiConnection
-	 */
-	function getConnection()
+	public function __construct(string $tempDir, DibiConnection $conection)
```

Comment:
only note: the latest version of dibi has Dibi\Connection insted of DibiConnection.. it would be nice to be compatible with latest dibi